README.md

```diff
@@ -1 +1 @@
-# Final-Project
+# GoKit-Project-Bootcamp
```

Comment:
you can add a descriptiont of the project, what is about logic tecnologies used in, how to deploy, where de .sql is (this is a boot camp you can provide and .sql file with the DB shema)